server/controllers/filestore.controller.js

```diff
@@ -21,9 +21,39 @@ const resTypeMap = {
     "html": "html",
     "xml": "xml"
 };
+const fileTypeFunctionMap = {
+    "csv": "readCsvFile"
+}
 
 class FileStoreController {
 
+    readCsvFile(absolutePath) {
+        return new Promise(function (resolve, reject) {
+            let resultArr = []
```

Comment:
semicolon should be present in line 32, 33 
in line 34 the '.' after csv() should come in line 35 before fromFile